pkg/client/logs.go

```diff
@@ -187,14 +187,12 @@ func watchPodsToStreamLogs(client kubernetes.Interface, cfg *LogConfig, podCh ch
 
 	go func() {
 		for {
-			select {
-			case v := <-ch:
-				if v.Type == watch.Added && v.Object != nil {
-					switch t := v.Object.(type) {
-					case *v1.Pod:
-						podCh <- t
-					default:
-					}
+			v := <-ch
+			if v.Type == watch.Added && v.Object != nil {
+				switch t := v.Object.(type) {
+				case *v1.Pod:
+					podCh <- t
+				default:
```

Comment:
@vladimirvivien `gosimple` flags `for { select {} }` as redundant here. I _think_ this should preserve the original behavior, since `v := <-ch` is a blocking call on either case, correct?

Reply:
I am guessing there is only one channel in that select block and that's why it's flagged. Sure.